File: overview/rev1/guides/orders/php/simple/customer-create.php
<?php
$marketplace = \Balanced\Marketplace::mine();
$merchant = $marketplace->customers->create(array(
    "dob_month" => "7",
    "dob_year" => "1963",
    "name" => "Henry Ford",
    "address" => array(
        "postal_code" => "48120",
    )
));

$order = $merchant->orders->create();
?>

File: overview/rev1/guides/orders/csharp/simple/customer-create.cs
Customer merchant = Customer.new();
Dictionary<string, string> address = new Dictionary<string, string>();
address.Add("postal_code", "48120");
customer.name = "Henry Ford";
customer.dob_month = "7";
customer.dob_year = "1963";
customer.address = address;
customer.Save();